lib/jekyll.rb

```diff
@@ -115,9 +116,11 @@ def configuration(override = Hash.new)
     # timezone - the IANA Time Zone
     #
     # Returns nothing
+    # rubocop:disable Style/AccessorMethodName
     def set_timezone(timezone)
-      ENV['TZ'] = timezone
+      ENV["TZ"] = timezone
     end
+    # rubocop:enable Style/AccessorMethodName
```

Comment:
I don't think this is necessary 😄  It's just applies to the method when it's right above the method like that, I'm pretty sure.